static/app/components/smartSearchBar/index.tsx

```diff
@@ -1139,16 +1246,29 @@ class SmartSearchBar extends Component<Props, State> {
     const queryCharsLeft =
       maxQueryLength && query ? maxQueryLength - query.length : undefined;
 
-    this.setState(
-      createSearchGroups(
-        searchItems,
-        hasRecentSearches ? recentSearchItems : undefined,
-        tagName,
-        type,
-        maxSearchItems,
-        queryCharsLeft
-      )
+    const searchGroups = createSearchGroups(
+      searchItems,
+      hasRecentSearches ? recentSearchItems : undefined,
+      tagName,
+      type,
+      maxSearchItems,
+      queryCharsLeft
+    );
+
+    const commonActionsSearchResults = getQuickActionsSearchGroup(
```

Comment:
nit: should we just call this `quickActionsSearchResults` instead of `commonActionsSearchResults` for consistency?